autopush/endpoint.py

```diff
@@ -12,6 +13,13 @@
 from twisted.python import log
 
 
+def MakeEndPoint(fernet, endpoint_url, uaid, chid):
```

Comment:
That it's a pretty good idea, reduces the argument list, and makes things lemony fresh?